HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorOrder.json

```diff
@@ -0,0 +1,529 @@
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "drop-down",
+                    "display": "Drop down"
+                  }
+                ],
+                "text": "Drop down"
+              }
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorOrderPrepopulation\".DeviceRequestHcpcsCoding"
+              }
+            }
+          ],
+          "required": true,
+          "linkId": "5.1",
+          "text": "Glucose Monitor Code",
+          "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.84"
+        },
+        {
+          "type": "choice",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "drop-down",
+                    "display": "Drop down"
+                  }
+                ],
+                "text": "Drop down"
+              }
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorOrderPrepopulation\".MonitorType"
+              }
+            }
+          ],
+          "required": true,
+          "linkId": "5.2",
+          "text": "Glucose Monitor Type",
+          "answerOption": [
+            {
+              "valueCoding": {
+                "code": "Standard Monitor",
+                "display": "Standard Monitor"
+              }
+            },
+            {
+              "valueCoding": {
+                "code": "Monitor With Special Features",
+                "display": "Monitor With Special Features"
+              }
+            }
+          ]
+        },
+        {
+          "type": "string",
+          "linkId": "5.3",
+          "required": true,
+          "text": "Description (or brand name and model number) of Glucose Monitor"
+        },
+        {
+          "type": "choice",
+          "required": true,
+          "linkId": "5.4",
+          "text": "Is patient or caregiver able to use the glucose monitor?",
+          "answerValueSet": "http://terminology.hl7.org/ValueSet/v2-0136"
+        }
+      ]
+    },
+    {
+      "type": "group",
+      "linkId": "6",
+      "text": "Frequency of Use",
+      "item": [
+        {
+          "type": "group",
+          "linkId": "6.1",
+          "text": "Non-insulin Treated Frequency",
+          "enableWhen": [
+            {
+              "question": "4.2",
+              "operator": "!=",
```

Comment:
Need LHC form 25.1.2 (or later) to work

Reply:
That was the intention. Perhaps we should get a PR into DTR before this is merged.

Reply:
Can you create PR for that?

Reply:
Will do.

Reply:
PR Created. https://github.com/HL7-DaVinci/dtr/pull/106